### PR TITLE
Add stakeholder SIGs for wg-apply

### PR DESCRIPTION
**Pull request by zjs**
Add SIG API Machinery and SIG CLI as the stakeholder SIGs for the Apply working group based on the [working group's meeting notes](https://docs.google.com/document/d/1j_UqnJRvB7AywfZauHybVVNCOoDDA1AGWGxzPGE08ko/edit).

Additionally, update the meeting frequency for the Apply working group.

Towards #2176.

/kind documentation
/sig contributor-experience
/wg apply
/cc @lavalamp

### CODE REVIEW

File: sigs.yaml [hunk 1]
```diff
@@ -2276,6 +2276,8 @@ workinggroups:
 
       Resources can be found in [this Google drive folder](https://drive.google.com/drive/folders/1wlpgkS2gFZXdp4x2WlRsfUBxkFlt2Gx0)
     charter_link:
+    stakeholder_sigs:
+    - CLI
```
Comment (nikhita) [inline sigs.yaml]: API Machinery should also be here.

Comment (lavalamp) [reply sigs.yaml]: +1

File: wg-apply/README.md [hunk 2]
```diff
@@ -11,6 +11,9 @@ To understand how this file is generated, see https://git.k8s.io/community/gener
 Improve the state of declarative object management by fixing `kubectl apply`, moving the logic from the client to the server. This move also enables new/fixed features included in the scope of this working-group: diff, dry-run and prune.
 Resources can be found in [this Google drive folder](https://drive.google.com/drive/folders/1wlpgkS2gFZXdp4x2WlRsfUBxkFlt2Gx0)
 
+## Stakeholder SIGs
+* SIG CLI
```
Comment (lavalamp) [inline wg-apply/README.md]: Add API Machinery, or omit this section completely (since it's in metadata already).

Comment (lavalamp) [reply wg-apply/README.md]: Oh wait, this is a different document. Yes, please add API Machinery.

File: wg-apply/README.md [hunk 3]
```diff
@@ -11,6 +11,9 @@ To understand how this file is generated, see https://git.k8s.io/community/gener
...
+
 ## Meetings
 * Regular WG Meeting: [Tuesdays at 9:30 PT (Pacific Time)](https://docs.google.com/document/d/1FQx0BPlkkl1Bn0c9ocVBxYIKojpmrS1CFP5h0DI68AE/edit) (weekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=9:30&tz=PT%20%28Pacific%20Time%29).
```
Comment (lavalamp) [inline wg-apply/README.md]: We've switched to every other tuesday...

File: sigs.yaml (hunk 1 above)
Comment (zjs) [reply sigs.yaml]: Fixed in ae4388c.

File: wg-apply/README.md (hunk 2 above)
Comment (zjs) [reply wg-apply/README.md]: Fixed in ae4388c.

(It's not obvious, but this file is generated from `sigs.yaml`; adding API Machinery to the `stakeholder_sigs` list causes it to be included in this `README.md` automatically.)

File: wg-apply/README.md (hunk 3 above)
Comment (zjs) [reply wg-apply/README.md]: Fixed in 484c683.

**Comment by nikhita** [comment]
Looks good! :+1: 

@zjs could you also squash the commits into a single commit? Thanks!

**Comment by zjs** [comment]
Squashed (and rebased), sorry for the delay!

**Comment by nikhita** [comment]
/hold cancel
/lgtm

Thanks for squashing, Zach!

**Pull request merged**